File: Python/1010 - Cálculo Simples.py
P1,NP1,VUP1 = map(float,input().split( ))
P2,NP2,VUP2 = map(float,input().split( ))

soma1 = NP1*VUP1
soma2 = NP2*VUP2
somatotal = soma1 + soma2

print(f"VALOR A PAGAR: R$ {somatotal:.2f}")

File: JavaScript/1010 - Cálculo Simples.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8').split('\n');

var [P1, NP1, VUP1] = input[0].split(' ').map(parseFloat);
var [P2, NP2, VUP2] = input[1].split(' ').map(parseFloat);

var soma1 = NP1 * VUP1;
var soma2 = NP2 * VUP2;
var somatotal = soma1 + soma2;

console.log(`VALOR A PAGAR: R$ ${somatotal.toFixed(2)}`);